KarhooUISDK/Screens/CheckoutScreen/Components/BottomSheets/Comments/KarhooBottomSheetCommentsViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  KarhooBottomSheetCommentsViewModel.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 18/01/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+final class KarhooBottomSheetCommentsViewModel: ObservableObject {
+    @Published var textViewText: String
+
+    let initialValueForTextView: String
+    let onSaveCallback: (_ newComment: String) -> Void
+
+    let viewSubtitle: String
+    let buttonText = UITexts.Generic.save
+    
+    init(
+        initialValueForTextView: String,
+        viewSubtitle: String,
+        onSaveCallback:  @escaping (_ newFlightNumber: String) -> Void
+    ) {
+        self.initialValueForTextView = initialValueForTextView
+        self.viewSubtitle = viewSubtitle
+        self.onSaveCallback = onSaveCallback
+        textViewText = initialValueForTextView
```

Comment:
In init `self.` should be added everywhere is its applicable. Just to make it bodywise, unified.

Reply:
If method parameter and object variable name are different then self is not required (it is presenter as a warning if self is unnececery)